frontend/src/main/scala/bloop/engine/ExecutionContext.scala

```diff
@@ -13,7 +13,15 @@ import monix.execution.{ExecutionModel, UncaughtExceptionReporter}
 import monix.execution.schedulers.ExecutorScheduler
 
 object ExecutionContext {
-  private[bloop] val nCPUs = Runtime.getRuntime.availableProcessors()
+  private[bloop] val nCPUs = {
+    val default = Runtime.getRuntime.availableProcessors()
+    Option(System.getProperty("bloop.computation.cores")) match {
```

Comment:
Can we also try an environment variable? For example, I’m not sure how I would pass a system property to the bloop launcher.

Reply:
You would pass it as a normal argument prepended with `-J`. Given that you will have control over the launcher in your CI, do you think you can work with that? I don’t like exposing an environment variable because:

a) the server is spawned in places where environment variables are scarce and sometimes not modifiable (think launchd)

b) it gives the impression that changing the environment variable would dynamically change the thread pool

c) As a matter of style, I’m avoiding environment variables to configure bloop. System properties are more adequate because they are passed exactly when the server is started

Reply:
Sure I can work with a system property